certbot/certbot/_internal/plugins/standalone.py

```diff
@@ -1,8 +1,8 @@
 """Standalone Authenticator."""
+# https://github.com/python/typeshed/blob/master/stdlib/2and3/socket.pyi
```

Comment:
Another comment that got reordered -- I can't seem to find an `isort` option to fix this sort of thing, we may just have to manually check through for these.

Reply:
Done.